### ISSUE TITLE
How to fine tune your model for another language dataset?

### ISSUE THREAD

**Issue by shagho** [Wendison/VQMIVC#17]
Hi
I'm work in your job that doing for voice conversion. for this I have a question.
How to fine tune your model for another language dataset?

**Comment by Wendison** [comment]
Hi, I haven't finetuned the model with another language, but based on my experience, I think you could directly finetune my released VC model with the target lauguage dataset, following the same procedures to prepare and preprocess the data to obtain the acoustic features, maybe you can try to use a smaller learning rate and vary the training steps, it depends on your dataset size. Besides, one useful way to evaluate whether the finutuning is successful is to listen to the intermediate converted results. Good luck :)

**Comment by shagho** [comment]
Thank you man.
